sw/host/opentitanlib/src/transport/dediprog/gpio.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{ensure, Result};
+use lazy_static::lazy_static;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use crate::collection;
+use crate::io::gpio::{GpioError, GpioPin, PinMode, PullMode};
+use crate::transport::dediprog::Inner;
+use crate::util::parse_int::ParseInt;
+
+pub struct DediprogPin {
+    inner: Rc<RefCell<Inner>>,
+    index: u8,
+}
+
+impl DediprogPin {
+    const LAST_PIN_NUM: u8 = 15;
+
+    pub fn open(inner: Rc<RefCell<Inner>>, pinname: &str) -> Result<Self> {
+        let _ = pinname;
```

Comment:
This `let` looks extraneous.

Reply:
It certainly does.  It must have been left over from before the body of this method was implemented.